package test02_try_catch;

public class ExceptionTest1 {

}
